### PR TITLE
GitHub OAuth

### PR DESCRIPTION
**Pull request by KrzysztofUrbanczyk**
Recreated.
Old PR #24

### CODE REVIEW

**Comment by KrzysztofUrbanczyk** [comment]
As @ProximaB mentioned in the previous PR, we should determine how secrets will be stored in our application.

File: Coyote/CoyoteNETCore.Runner/Startup.cs [hunk 1]
```diff
@@ -43,12 +41,20 @@ public void ConfigureServices(IServiceCollection services)
             {
                 c.SwaggerDoc("v1", new Info { Title = "Coyote API", Version = "v1" });
             });
+
+            services.AddAuthentication(CookieAuthenticationDefaults.AuthenticationScheme)
+                .AddCookie(options =>
+                {
```
Comment (dotNETSanta) [inline Coyote/CoyoteNETCore.Runner/Startup.cs]: What about configuring AccessDeniedPath here?

**Comment by silvuss** [comment]
> As @ProximaB mentioned in the previous PR, we should determine how secrets will be stored in our application.

@KrzysztofUrbanczyk This is a good point, but may you list the secrets that you think we will be using?

**Comment by KrzysztofUrbanczyk** [comment]
@silvuss We currently use two secrets:
- GitHub secrets ([link](https://github.com/CoyoteNET/CoyoteNET-backend/pull/28/files#diff-b8ab67bac232c2ee61f7b896d43e4ecdR5))
- ConnectionString ([link](https://github.com/CoyoteNET/CoyoteNET-backend/blob/master/Coyote/CoyoteNETCore.Runner/appsettings.json#L3))

Of course, for development we can use: `dotnet user-secrets` but everyone will have to configure them by themselves.

**Comment by dotNETSanta** [comment]
I'd rather go for appsettings during development.

**Pull request merged**